tests/contract-call.sh

```diff
@@ -4,36 +4,33 @@ set -eu -o pipefail
 
 # Examples:
 # contract-call.sh test_empty
-# contract-call.sh test_bool "true" "false"
+# contract-call.sh test_bool "[true, false]"
 
 CONTRACT_SRC="contracts/Test.aes"
 ADDRESS_SRC="build/${CONTRACT_SRC//.aes/.addr}"
 WALLET_PATH="build/wallet"
 WALLET_PASS="123"
-# init() calldata
-CALLDATA="cb_KxFE1kQfP4oEp9E="
 
-if ! aecli account address -P $WALLET_PASS $WALLET_PATH > /dev/null; then
-    ACCOUNT=$(aecli account create --json -P $WALLET_PASS $WALLET_PATH | jq -r '.publicKey')
+if ! npx aecli account address -P $WALLET_PASS $WALLET_PATH > /dev/null; then
+    ACCOUNT=$(npx aecli account create --json -P $WALLET_PASS $WALLET_PATH | jq -r '.publicKey')
     curl -sS -X POST https://faucet.aepps.com/account/$ACCOUNT > /dev/null
 fi
 
-if ! aecli account balance -P $WALLET_PASS $WALLET_PATH > /dev/null; then
+if ! npx aecli account balance -P $WALLET_PASS $WALLET_PATH > /dev/null; then
     echo "Please feed the wallet"
-    aecli account address -P $WALLET_PASS $WALLET_PATH
+    npx aecli account address -P $WALLET_PASS $WALLET_PATH
 fi
 
 if [ ! -f $ADDRESS_SRC ]; then
-    RESULT=$(aecli contract deploy --json -P $WALLET_PASS --networkId ae_uat $WALLET_PATH $CONTRACT_SRC $CALLDATA)
-    rm $(echo $RESULT | jq -r '.descPath')
```

Comment:
Any idea what was the role of `rm` that you've removed ?

Reply:
Yep, it is to remove wasteful contract descriptors generated by cli. I'm making this script to use descriptor param that you requested before, so `rm` shouldn't be needed.